src/server/controller/user/invite.ts

```diff
@@ -28,7 +28,8 @@ export const invite = async (
     // If user with primary email not found, check if user has active google login
     if (!userToInvite) userToInvite = await UserRepository.getOne({ emailGoogle: email }, t)
     // If neither of above, create new user
-    if (!userToInvite) userToInvite = await UserRepository.create({ user: { email, name: name ?? '' } }, client)
+    if (!userToInvite)
+      userToInvite = await UserRepository.create({ user: { email, props: { name: name ?? '' } } }, client)
```

Comment:
<img width="702" alt="image" src="https://user-images.githubusercontent.com/1138156/214827132-ee1762d1-4ed5-429f-bc41-eefc384ba719.png">

Reply:
solved. lang is mandatory.